src/emqttd_access_rule.erl

```diff
@@ -96,10 +84,7 @@ bin(L) when is_list(L) ->
 bin(B) when is_binary(B) ->
     B.
 
-%%------------------------------------------------------------------------------
-%% @doc Match rule
-%% @end
-%%------------------------------------------------------------------------------
+%% @doc Match Access Rule
 -spec match(mqtt_client(), topic(), rule()) -> {matched, allow} | {matched, deny} | nomatch.
 match(_Client, _Topic, {AllowDeny, all}) when (AllowDeny =:= allow) orelse (AllowDeny =:= deny) ->
```

Comment:
Line 89 is too long: match(_Client, _Topic, {AllowDeny, all}) when (AllowDeny =:= allow) orelse (AllowDeny =:= deny) ->.